docker-compose.demo.yml

```diff
@@ -94,3 +94,8 @@ services:
         command: --automigrate
         environment:
             - CREATE_ARTIFACT_SKIPVERIFY=1
+
+    mender-generate-delta-worker:
```

Comment:
this will prevent starting open source demo, because mender-generate-delta-worker is not an open source component

Reply:
Yes, this cannot go here. Generally speaking we don't have an "enterprise demo" mode. We only have this `--enterprise-testing` flag that we use internally (hence it is not documented).

As this is only needed for tests, it should go into [`docker-compose.testing.enterprise`](https://github.com/mendersoftware/integration/blob/master/docker-compose.testing.enterprise.yml) instead.

Reply:
yes, and also breaks the os test, I also forgot that this is ent only. fixed. sorry for that.